docs/modules/migrate/pages/dmt-command-reference.adoc

```diff
@@ -43,6 +43,11 @@ Parameters:
 |If set, the migration starts immediately, without any confirmation.
 |`false`
 
+|`--config`
+|Required
+|Note that this can be omitted if a named config is added before and there is only a single named config DMT can choose.
```

Comment:
```suggestion
|This can be omitted if a named config has been added and only a single, named, config is available.
```

Reply:
Two questions:

1. Can't it be omitted if multiple configs have been added - you just need to select the required config when prompted?
2. If using `--config`, doesn't that make the `[dmt-config-path]` option redundant in the original Usage example?

Reply:
1. yes let me change
2. dmt-config-path is for a different config as you know. so why should it make it redundant? do you mean it creates confusion?

Reply:
Yes, it would be good if some broader explanation could be added to avoid any possible confusion.

Reply:
I added description